tests/e2e/operator_ha_test.go

```diff
@@ -42,6 +42,9 @@ var _ = Describe("Operator High Availability", Serial,
 			if testLevelEnv.Depth < int(level) {
 				Skip("Test depth is lower than the amount requested for this test")
 			}
+			if IsGKE() {
```

Comment:
I think we should add a common interface for this instead of hardcoding it to GKE

Reply:
good idea. Doing it.